airdrop/airdrop.gno

```diff
@@ -35,19 +37,21 @@ type Config struct {
 
 // Airdrop represents the airdrop contract
 type Airdrop struct {
-	token         grc20.Token
-	config        Config
-	claimedBitmap map[uint64]uint64
-	address       std.Address
+	token           grc20.Token
+	config          Config
+	claimedBitmap   map[uint64]uint64
```

Comment:
Please add meaningful comments for map something like this
```
map[uint64]uint64 // bitMap -> balance

Reply:
https://github.com/gnoswap-labs/governance/pull/8/commits/029f047b24d5fdbe956f5fa7d2975ed926435480